server/src/main/java/org/elasticsearch/search/aggregations/ParsedAggregation.java

```diff
@@ -36,7 +36,7 @@
  */
 public abstract class ParsedAggregation implements Aggregation, ToXContentFragment {
 
-    protected static void declareAggregationFields(ObjectParser<? extends ParsedAggregation, Void> objectParser) {
+    protected static void declareAggregationFields(AbstractObjectParser<? extends ParsedAggregation, ?> objectParser) {
```

Comment:
I'm using a `ConstructingObjectParser` to build `ParsedTopHits`, thus the change here.